test/Microsoft.AspNet.Mvc.Core.Test/Formatters/OutputFormatterTests.cs

```diff
@@ -50,8 +50,8 @@ public static IEnumerable<object[]> SelectResponseCharacterEncodingData
             // Arrange
             var mockHttpContext = new Mock<HttpContext>();
             var httpRequest = new DefaultHttpContext().Request;
-            httpRequest.Headers["Accept-Charset"] = acceptCharsetHeaders;
-            httpRequest.ContentType = "application/acceptCharset;charset=" + requestEncoding;
+            httpRequest.Headers[HeaderNames.AcceptCharset] = acceptCharsetHeaders;
+            httpRequest.Headers[HeaderNames.Accept] = "application/acceptCharset;charset=" + requestEncoding;
```

Comment:
The formatter used to choose the encoding from the request `ContentType` which was removed in this PR. I'll change `context.ContentType` to be `application/acceptCharset;`